src/styles/_typography.scss

```diff
@@ -1,53 +1,41 @@
-// Styles that go into this file should apply generally to the entire site. Specific component typography should be contained in components.
-
-// Headings
-h1,
-h2,
-h3,
-h4 {
-  // Spacing
-  --flow-spacing: var(--gcds-spacing-600);
```

Comment:
I added this spacing to have more space before a heading, and only apply when "flow" is a class of the container. Its nice to have a bit more space before headings to keep them closer to the content right under it. 

This works like the `space-y`class from tailwind. In other places, the variable won't have an impact.

Reply:
Well, you had the flow class applied to the main container which means that these flow classes were applied everywhere. Since we are trying to show the utility framework in action, we should be using the utility framework for things like this. It's impossible to overwrite these flow classes with the utility framework and it doesn't make sense to add extra CSS to overwrite them.

Reply:
Sounds good